src/FFMpeg/Media/Waveform.php

```diff
@@ -66,6 +75,50 @@ public function addFilter(WaveformFilterInterface $filter)
     }
 
     /**
+     * Parameter should be an array containing at least one valid color represented as a HTML color string. For
+     * example #FFFFFF or #000000. By default the color is set to black. Keep in mind that if you save the waveform
+     * as jpg file, it will appear completely black and to avoid this you can set the waveform color to white (#FFFFFF).
+     * Saving waveforms to png is strongly suggested.
```

Comment:
Please insert a new paragraph before this line.

Reply:
Done